tests/test_plot.py

```diff
@@ -692,31 +692,92 @@ def test_image_plot_ratio_hist():
     return fig
 
 
+def _make_plot_ratio_callable_test_figure(hist_axis, n_numbers):
+    """
+    Helper function to create a plot_ratio figure with a gaussian model.
+
+    The purpose is to create tests for histograms with different hist_axes, e.g.
+    regular and various variable binnings.
+    """
+    rng = np.random.default_rng(42)
```

Comment:
Here I used a random number generator because `np.random.seed` is officially the recommended way to generate random numbers with a fixed seed, its current documentaition says:

> The best practice is to **not** reseed a BitGenerator, rather to
> recreate a new one. This method is here for legacy reasons.

**However**, I just noticed that the random seed not only affects the numbers to fill the histogram with but also the fit which uses `np.random.normal` [here](https://github.com/meliache/hist/blob/feature/allow-for-irregular-pull-widths/src/hist/plot.py#L347), so this might explain why some tests fail. I think I need to change this back to `np.random.seed(42)`.